app/src/main/java/com/moodup/movies/ui/authentication/LoginFragment.kt

```diff
@@ -62,6 +65,11 @@ class LoginFragment : Fragment() {
                 AuthLoginState.ON_LOGIN_FAILURE->{
                     onLoginFailure()
                 }
+
+                AuthLoginState.ON_ALREADY_LOGGED_IN->{
+                    context?.let { viewModel?.startMainActivity(it) }
```

Comment:
Nie powinniśmy przekazywać kontekstu do viewModelu więc startMainActivity() powinno być we fragmencie.